tests/generated/wasi/clocks/monotonic-clock/empty.s

```diff
@@ -0,0 +1,3 @@
+// This file exists for testing this package without WebAssembly,
```

Comment:
We prohably don't want to commit the generated Go code?

Reply:
I think it's useful to have the generated code committed for gopls/IDE and to show how generated code changes over time.